cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -410,6 +374,86 @@ describe('User index view', () => {
               }).should('exist');
             });
           });
+
+          it('Opens the add organization modal', () => {
+            cy.enableFeatureFlag('member_index_view');
+            // Helper function for cypress-pipe
+            const click = (el) => el.click();
+
+            cy.findByRole('button', { name: 'User actions: Admin McAdmin' })
+              .as('userActionsButton')
+              .pipe(click)
+              .should('have.attr', 'aria-expanded', 'true');
```

Comment:
I don't think we need to do this now, but as we go on adding tests that click things in this dropdown, it might be useful to bring this whole chunk out into a helper function so we can just call e.g. `openUserActionsDropdown()`